tests/integration/cli/tests/config.rs

```diff
@@ -65,9 +67,9 @@ fn wasmer_config_error() -> anyhow::Result<()> {
         .map(|s| s.trim().to_string())
         .collect::<Vec<_>>();
     #[cfg(not(windows))]
-    let expected_1 = "Usage: wasmer config --bindir --cflags";
+    let expected_1 = "Usage: wasmer-cli-shim config --bindir --cflags";
```

Comment:
Seeing different command names opposed to the  the proper `wasmer` command would be somewhat confusing, wouldn't it?
Needs to be at least documented, maybe in the README.